projects/ngx-editor/src/lib/modules/menu/toggle-command/toggle-command.component.ts

```diff
@@ -56,7 +56,10 @@ export class ToggleCommandComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
-    this.html = Icon.get(this.name);
+    const icon = this.ngxeService.config.icons[this.name]
+      ? this.ngxeService.config.icons[this.name]
+      : Icon.get(this.name);
```

Comment:
```suggestion
    const icon = this.ngxeService.config.icons[this.name]
      ? this.ngxeService.config.icons[this.name]
      : Icon.get(this.name);
```

May be a method that does this logic inside the service. `this.ngxeService.getIcon(<icon_name>)`, should resolve correctly either with given config or default. to keep the code DRY for future uses.